content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -45,57 +49,57 @@ Use this guide to manually set up the Datadog [AWS Integration][1].
-16. Click `Next: Tags` and `Next: Review`.
-17. Give the role a name such as `DatadogIntegrationRole`, as well as an apt description. 
-18. Click `Create Role`.
+8. Create a new role in the AWS [IAM Console][3].
+9. Select **AWS account** for the trusted entity type, and **Another AWS account**.
+10. Enter `464622532012` as the `Account ID`. This is Datadog's account ID, and grants Datadog access to your AWS data.
+11. Select **Require external ID** and enter the external ID copied in the [Generate an external ID](#generate-an-external-id) section. 
+Ensure to leave `Require MFA` disabled. 
+12. Click **Next**.
+13. If you've already created the policy, search for it on this page and select it. Otherwise, click **Create Policy**, which opens in a new window, and follow the instructions from the previous section. 
+14. Optionally, attach the <a href="https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/SecurityAudit" target="_blank">AWS SecurityAudit Policy</a> to the role to use Datadog's [Cloud Security Posture Management product][4] (CSPM).
+15. Click **Next**.
+16. Give the role a name such as `DatadogIntegrationRole`, as well as an apt description. 
+17. Click **Create Role**.
 
 ### Complete the setup in Datadog
 
-19. Returning to the AWS integration tile page in your Datadog account that you had open in another tab, enter your AWS Account ID **without dashes**, for example: `123456789012`. Your Account ID can be found in the ARN of the role created for Datadog.
-20. Enter the name of the created role.  
-**Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name created on the AWS side.
-21. If there is a [Datadog is not authorized to perform sts:AssumeRole][5] error, make sure your AWS trust policy's `sts:ExternalId:` matches the `AWS External ID` previously created in the integration tile.
-22. Click **Install Integration**.
-23. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box <a href="https://app.datadoghq.com/screen/integration/7/aws-overview" target="_blank">AWS Overview Dashboard</a> to see metrics sent by your AWS services and infrastructure.
+18. Return to the AWS integration tile page in your Datadog account that you had open in another tab, and click the checkbox to confirm the Datadog IAM role was added to the AWS account.
+19. Enter the account ID **without dashes**, for example: `123456789012`. Your Account ID can be found in the ARN of the role created for Datadog.
+20. Enter the name of the role created in the previous section, and click **Save**.
+**Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name in AWS.
```

Comment:
```suggestion
    **Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name in AWS.
```

Reply:
Thanks for pointing this out as well, updated so that the note starts on a new line.